readme.txt

```diff
@@ -1,36 +1,70 @@
-=== Font Awesome ===
-Contributors: fontawesome, mlwilkerson, kemitchell
+=== Font Awesome Official ===
+Contributors: fontawesome, mlwilkerson, kemitchell, robmadole
```

Comment:
The readme validator complained that one of these is not like the other. Looks like `robmadole` is not a valid wordpress.org userid. If so, there would be a profile page for it here:
https://profiles.wordpress.org/robmadole

Like this [profile page](https://profiles.wordpress.org/bfintal) for one of the Stackable devs.
https://profiles.wordpress.org/mlwilkerson